SU-SWS/saa_victoria_subtheme
# ADAPT-715: Removed bottom padding from old test, set to padding-bottom 0

# READY FOR REVIEW

# Summary
- Removed old test line of padding, not needed and causes issues

# Review By (Date)
- 9/18 TODAY (launch monday morning)

# Urgency
- High

# Steps to Test

1. Pull down the branch or check out https://saastudents-test.sites.stanford.edu/faux-card-testing-page and https://saastudents-test.sites.stanford.edu/student-alumni-council 
2. Clear the site cache
3. Make sure that there isn't a big padding on the bottom of the image div wrapper. It should not be there covering the footer links.
You can see what the issue looks like here: https://students.alumni.stanford.edu/student-alumni-council
Where you can't click on the map or other footer links because the padding-bottom is covering it.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-715
